internal/tiltfile/tiltfile_state.go

```diff
@@ -1133,14 +1131,19 @@ func (s *tiltfileState) defaultedPortForwards(pfs []model.PortForward) []model.P
 	return result
 }
 
-// checkForImpossibleLiveUpdates logs a warning if the group of image targets contains
-// any impossible LiveUpdates (or FastBuilds).
-//
-// Currently, we only collect container information for the first Tilt-built container
-// on the pod (b/c of how we assemble resources, this corresponds to the first image target).
-// We won't collect container info on any subsequent containers (i.e. subsequent image
-// targets), so will never be able to LiveUpdate them.
-func (s *tiltfileState) checkForImpossibleLiveUpdates(m model.Manifest) error {
+func (s *tiltfileState) validateLiveUpdates(manifests []model.Manifest) error {
```

Comment:
this name seems pretty confusing next to `validateLiveUpdate`, which operates on a target

`validateManifestsLiveUpdates`? `validateLiveUpdatesForManifests`?

oy